community/neo4j/src/docs/ops/upgrades.asciidoc

```diff
@@ -4,89 +4,77 @@
 :manual-base-url: http://neo4j.com/docs/{neo4j-version}
 :manual-ha-upgrade-guide: {manual-base-url}/ha-upgrade-guide.html
 :manual-cypher-compatibility: {manual-base-url}/cypher-compatibility.html
+:neo4j-releases-download-page: http://neo4j.com/download/other-releases
 
 [IMPORTANT]
-This section describes upgrading a single Neo4j instance.
-To upgrade a Neo4j HA cluster (Neo4j Enterprise) a very specific procedure must be followed.
+====
+This section describes upgrading a _single_ Neo4j instance.
+
+To upgrade a _Neo4j HA cluster (Neo4j Enterprise)_, a very specific procedure must be followed.
 Please see
 ifndef::upgradetext[<<ha-upgrade-guide>>.]
 ifdef::upgradetext['Upgrade of a Neo4j HA Cluster' at {manual-ha-upgrade-guide}.]
+====
 
-Each Neo4j version supports upgrading from a limited number of previous versions.
-These upgrades are either automatic, or require explicit configuration to allow them.
-Upgrades between patch releases within the same minor version are automatic.
-Upgrading to the latest patch release is recommended before upgrading to new major or minor versions.
-
-A database created by an older version of Neo4j will be upgraded during startup when opened by the target version of Neo4j.
-
-The following Neo4j versions, where _x_ stands for the latest patch release, can be upgraded to Neo4j {neo4j-version}:
-
-1.9.x -> {neo4j-version}::
-<<explicit-upgrade,Explicit configuration is required>>
-
-2.0.x -> {neo4j-version}::
-<<explicit-upgrade,Explicit configuration is required>>
-
-2.1.x -> {neo4j-version}::
-<<explicit-upgrade,Explicit configuration is required>>
-
-2.2.x -> {neo4j-version}::
-<<explicit-upgrade,Explicit configuration is required>>
+[CAUTION]
+====
+Before starting any upgrade actions, it is very important to make a backup of the Neo4j database.
+If using the 
+ifndef::upgradetext[<<operations-backup, online backup tool>>, ]
+ifdef::upgradetext[online backup tool (see http://neo4j.com/docs/{neo4j-version}/operations-backup.html), ]
+available with Neo4j Enterprise, ensure that backups have completed successfully.
+====
 
 [NOTE]
-Downgrade is only supported between Neo4j versions that allow for automatic store upgrades.
-This typically means only within patch releases of the same Neo4j version.
-
-[[automatic-upgrade]]
-== Automatic Store Upgrade
-
-To perform an automatic store upgrade:
-
-. Cleanly shut down the older version of Neo4j, if it is running.
+====
+Throughout this instruction, the files used to store the Neo4j data are referred to as `database files`.
+The location of these files is specified by configuring the `org.neo4j.server.database.location` variable in `conf/neo4j-server.properties`.
+====
```

Comment:
Get me to help you with merging this forward to 3.0. the setting and config filename have both changed.